suave/builder/builder.go

```diff
@@ -75,3 +82,9 @@ func (b *builder) AddTransaction(txn *types.Transaction) (*types.SimulateTransac
 
 	return result, nil
 }
+
+func (b *builder) Terminate() {
+	b.cancelFunc()
+	b.txns = nil
```

Comment:
Why do you need to set this to nil?

Reply:
Misunderstood how context works here, fixed it.